analyzers/tests/SonarAnalyzer.UnitTest/TestCases/SupplyParameterFromQueryNeedsRoutableComponent_Noncompliant.cs

```diff
@@ -1,14 +0,0 @@
-﻿using Microsoft.AspNetCore.Components;
-using System;
-
-class SupplyParameterFromQueryNeedsRoutableComponent_Noncompliant : ComponentBase
```

Comment:
Some test files of S6803 with the previous name are still around:
- `SupplyParameterFromQueryNeedsRoutableComponent_Compliant.razor`
- `SupplyParameterFromQueryNeedsRoutableComponent_Partial.cs`
- `SupplyParameterFromQueryNeedsRoutableComponent_Partial.razor`